atlasdb-jepsen-tests/src/main/java/com/palantir/atlasdb/jepsen/events/Event.java

```diff
@@ -48,6 +50,7 @@ static Event fromKeywordMap(Map<Keyword, ?> map) {
     static Map<Keyword, Object> toKeywordMap(Event event) {
         Map<String, Object> rawStringMap = OBJECT_MAPPER.convertValue(event, new TypeReference<Map<String, ?>>() {});
         return EntryStream.of(rawStringMap)
+                .filterValues(Objects::nonNull)
```

Comment:
Okay. This can be done using `DeserializationFeature.FAIL_ON_NULL_FOR_PRIMITIVES`